ocs_ci/utility/vsphere_nodes.py

```diff
@@ -85,6 +85,31 @@ def restart_chrony(self):
             _rc = True
         return _rc
 
+    def set_host_name(self, host_name):
+        """
+        Sets the host name
+
+        Args:
+            host_name (str): Name to set as host name
+
+        Returns:
+            tuple: tuple which contains command return code, output and error
+
+        """
+        cmd = f"sudo hostnamectl set-hostname {host_name}"
+        return self.vmnode.exec_cmd(cmd)
+
+    def reboot(self):
+        """
+        Reboots the node
+
+        Returns:
+            tuple: tuple which contains command return code, output and error
```

Comment:
the same.  Do we need to wait somehow till the node is rebooted? Do we have some way to check if the vm got rebooted?

Reply:
This just issue reoboot and we are using retry at the caller function